x-pack/plugins/apm/public/components/app/ServiceMap/icons.ts

```diff
@@ -14,10 +14,12 @@ import {
 import { getAgentIcon } from '../../shared/AgentIcon/get_agent_icon';
 import { getSpanIcon } from '../../shared/span_icon/get_span_icon';
 
-export function iconForNode(node: cytoscape.NodeSingular) {
+export function iconForNode(node: cytoscape.NodeSingular, isDarkMode: boolean) {
```

Comment:
It looks like you are no longer using `isDarkMode`:
```suggestion
export function iconForNode(node: cytoscape.NodeSingular) {
```